docs/get_started.md

```diff
@@ -17,33 +17,29 @@ pip install graphrag
 ```
```

Comment:
Should we add a comment before this to the effect of "create your workspace folder and cd into it. We recommend creating a python virtual environment in this folder"?